config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
         # Change the value of image field below to your controller image URL
-        - image: eu.gcr.io/gardener-project/gardener/etcd-druid:v0.7.0-dev
+        - image: abdasgupta/etcd-druid:0.8.0-dev
```

Comment:
Same as above

Reply:
I would revert back this change. This change is automated

Reply:
Done